05.wc/wc.rb

```diff
@@ -50,11 +50,11 @@ def calculate_max_widths(total_stat)
   %i[lines words bytes].to_h { |key| [key, total_stat[key].to_s.length] }
 end
 
-def format_result(stats, max_widths, options)
+def format_row(stats, max_widths, options)
   result = %i[lines words bytes].filter_map do |key|
```

Comment:
```suggestion
  columns = %i[lines words bytes].filter_map do |key|
```

行に対して列ですよ、という変数名にすると少し具体性が上がって良いかも